src/Decorators/ArrayDecorator.php

```diff
@@ -23,17 +24,25 @@ class ArrayDecorator implements DecoratorInterface
      * @var \McCool\LaravelAutoPresenter\AutoPresenter
      */
     protected $autoPresenter;
+    /**
+     * The container instance.
+     *
+     * @var \Illuminate\Contracts\Container\Container
+     */
+    private $container;
 
     /**
      * Create a new array decorator.
      *
      * @param \McCool\LaravelAutoPresenter\AutoPresenter $autoPresenter
+     * @param \Illuminate\Contracts\Container\Container  $container
```

Comment:
I really did mean, please actually resolve the config in the service provider, and just pass the resolved config in here.